docs/production.md

```diff
@@ -93,8 +93,8 @@ After installing a Katello server, you could then spin up a Capsule with the ass
 
 For a release version in production:
 
-    ansible-playbook -l localhost playbooks/katello_31.yml
+    ansible-playbook -i "localhost," -c local playbooks/katello.yml
 
 For nightly production:
 
-    ansible-playbook -l localhost playbooks/katello_nightly.yml
+    ansible-playbook -i "localhost," -c local playbooks/katello_nightly.yml
```

Comment:
This playbook no longer exists either. Nightly is the default for ```playbooks/katello.yml``` if unspecified.